packages/indexer/src/main.ts

```diff
@@ -1,19 +1,18 @@
 import * as core from './core';
 import * as api from './api';
 import { DB } from './core/db';
-import { BunyanLevel, createLogger } from './core/utils';
+import { createLogger } from './core/utils';
 import { IndexerCollector } from './core/metrics';
 import { getSdk } from './core/sdk';
 import { startTokenUpdater } from './tokens';
-
-export type NomadEnvironment = 'development' | 'staging' | 'production';
-export type Program = 'api' | 'core';
-
-const environment = process.env.ENVIRONMENT as NomadEnvironment;
-const configOverrideLocation = process.env.CONFIG_OVERRIDE_LOCATION;
-const program = process.env.PROGRAM as Program;
-const logLevel = (process.env.LOG_LEVEL || 'debug') as BunyanLevel;
-const metricsPort = parseInt(process.env.METRICS_PORT || '9090');
+import {
+  program,
```

Comment:
program vs Program?

Reply:
Program is the enum that contains multiple variations of what program (small p) could be